<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRecordsettings">
      <attribute name="title">
       <string>Record settings (ffmpeg)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelRecordheader">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Record settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelfps">
           <property name="text">
            <string>Frames pr. second</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxfps"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelvideocodec">
           <property name="text">
            <string>Videocodec</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditvideocodec"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelaudiocodec">
           <property name="text">
            <string>Audiocodec</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditaudiocodec"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelaudiochannles">
           <property name="text">
            <string>Audio channels</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBoxaudiochannels"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelRecordingdevice">
           <property name="text">
            <string>Recording device</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxrecording"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEditbasename"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelbasename">
           <property name="text">
            <string>Basename</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labelpath">
           <property name="text">
            <string>Default path</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEditpath">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonpathBrowse">
             <property name="text">
              <string>Change</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="labelFormat">
           <property name="text">
            <string>Default format</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="comboBoxFormat"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>49</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>labelRecordheader</zorder>
      <zorder>verticalSpacer</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonRestore">
       <property name="text">
        <string>Restore to default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
